sql/core/src/test/resources/sql-tests/inputs/datetime.sql

```diff
@@ -164,3 +164,6 @@ select from_csv('26/October/2015', 'date Date', map('dateFormat', 'dd/MMMMM/yyyy
 select from_unixtime(1, 'yyyyyyyyyyy-MM-dd');
 select date_format(timestamp '2018-11-17 13:33:33', 'yyyyyyyyyy-MM-dd HH:mm:ss');
 select date_format(date '2018-11-17', 'yyyyyyyyyyy-MM-dd');
+
+-- SPARK-31879: the first day of week
+select to_timestamp('2020-01-01', 'YYYY-ww-uu');
```

Comment:
can we use formatting in the test? so this doesn't conflict with https://github.com/apache/spark/pull/28706